### ISSUE TITLE
Update documentation for RHEL7

### ISSUE THREAD

**Issue by ericfranz** [OSC/ood-documentation#1]
1. update https://osc.github.io/ood-documentation/installation/modify-system-security.html?highlight=iptables to include firewalld
2. ???

**Comment by brianmcmichael** [comment]
First Installation on RHEL7 is done. @nickjer is talking to systems presently about some issues with passenger processes being unexpectedly killed. It may be an SSL thing. Install notes to follow.

**Comment by brianmcmichael** [comment]
## 1. Software Requirements

For RHEL 7, the subscription manager command is:

```
sudo subscription-manager repos --enable=rhel-server-rhscl-7-rpms
```

I experienced some artifacts in Chrome while triple-clicking the Sphinx documentation. @nickjer was able to replicate in Chrome linux. This does not appear in Firefox.

Whenever a `<pre>` is at the bottom of the page, triple-clicking the line copies the Labels from the navigation buttons.

```
mkdir -p ~/ood/src
Next  Previous
```

## 2. Modify System Security

Use `sudo /usr/sbin/service iptables save` instead of `sudo /etc/init.d/iptables save`

This may not be an issue if we use `firewalld` instead.

## 9. Install Applications

`cd ~/ood/src/sys`

This dir does not exist. Need a `mkdir` command before this.

## 9.6 My Jobs

Need to add the `git19` scl to the bundle install command since it uses ood-job from github.

Should be:
```
scl enable rh-ruby22 git19 -- bin/bundle install --path vendor/bundle
```

Unless we move ood-job to rubygems.

Also got the Sphinx issue on that page:

```
sudo cp -r . /var/www/ood/apps/sys/myjobs
Next  Previous
```

**Comment by nickjer** [comment]
Added the `sudo subscription-manager ...`
Fixed 9
Fixed 9.6

Ignoring the Sphinx/Chrome issue for now.
Will request firewalld.
